doc/en/how-to/fixtures.rst

```diff
@@ -732,12 +732,64 @@ Here's how the previous example would look using the ``addfinalizer`` method:
 It's a bit longer than yield fixtures and a bit more complex, but it
 does offer some nuances for when you're in a pinch.
 
+Note on finalizer order
+""""""""""""""""""""""""
+
+Finalizers are executed in a first-in-last-out order, while operations after yield are executed sequentially.
+Consider the differences in the following examples:
+
+.. code-block:: python
+
+    import pytest
+    from functools import partial
+
+
+    @pytest.fixture
+    def fix_w_finalizers(request):
+        request.addfinalizer(partial(print, "finalizer_2"))
+        request.addfinalizer(partial(print, "finalizer_1"))
+
+
+    @pytest.fixture
+    def fix_w_yield():
+        yield
+        print("after_yield_1")
```

Comment:
This is not a good example I think, because of course both prints will be executed sequentially (pytest cannot mess with how Python executes code within a function).

I suggest we use two fixtures:

```python
    @pytest.fixture
    def fix_w_yield1():
        yield
        print("after_yield_1")

    @pytest.fixture
    def fix_w_yield2():
        yield
        print("after_yield_2")
```

And then make `test_bar` request both:

```python
    def test_bar(fix_w_yield1, fix_w_yield2):
        print("test_bar")
```

Reply:
Ah, I see.
The execution order with the addfinalizer method is evident, so the execution order only needs clarification in this case.
Lovely! How about now?

Reply:
I think we can keep the examples using `addfinalizer` as they are useful for comparision, so could you please bring them back? Thanks

Reply:
Sure thing, re-added.